backend/main/src/Shared/Adapters/DB/NishikiDBTypes.ts

```diff
@@ -43,3 +43,58 @@ export type InvitationLink = {
 	linkExpiryTime: Date;
 	invitationLinkHash: string;
 };
+
+/**
+ * This is a food object structure recorded in the DB.
+ */
+export type FoodItem = {
+	FoodId: string;
+	Name: string | null;
+	Unit: string | null;
+	Quantity: number | null;
+	Category: string | null;
+	Expiry: string | null;
+	CreatedDatetime: string;
+};
+
+export type Food = {
+	foodId: string;
+	name: string | null;
```

Comment:
I think the type of Name should be string because the name is a required prop. Or is there any case it can be null if we use dynamoDB? ( same comment as another one)